packages/@lwc/template-compiler/src/__tests__/compileToFunction.spec.ts

```diff
@@ -93,6 +93,79 @@ it('should look up for rendering a component', () => {
     ]);
 });
 
+it('should transform spellcheck to boolean false when attr is "false"', () => {
```

Comment:
Please convert this compiler test to a static test (tests under the `fixtures` directory).